deployment/deploy.sh

```diff
@@ -18,35 +18,67 @@ source $CURRENT_DIR/vars/$DEPLOY_ENV.sh
 yarn global add serverless@1.32.0
 yarn
 
+# Create s3 bucket
+node $CURRENT_DIR/createS3Bucket.js
+if [ $? != 0 ]; then
+  echo "Bucket build failed"
+  exit 1
+fi
+
 # Deploy with serverless
 sls deploy -v | tee out.tmp
 if [ "${PIPESTATUS[0]}" != "0" ]; then
   echo "sls deploy failed"
   exit 1
 fi
 
+# Check if seeding will be required
+node $CURRENT_DIR/../db/scripts/writeSeedFlag.js
+if [ $? != 0 ]; then
+  echo "seed flag script failed"
+  exit 1
+fi
+
+# Initialize floods database
+node $CURRENT_DIR/../db/scripts/initialize.js
+if [ $? != 0 ]; then
+  echo "initialize db script failed"
+  exit 1
+fi
+
 # Run migrations
 export PGENDPOINT=$(grep "PgEndpoint" out.tmp | cut -f2- -d: | cut -c2-)
 export GRAPHQL_ENDPOINT=$(grep "GraphqlEndpoint" out.tmp | cut -f2- -d: | cut -c2-)
-node $CURRENT_DIR/../db/scripts/migrateAndSeed.js
+node $CURRENT_DIR/../db/scripts/migrate.js
 if [ $? != 0 ]; then
-  echo "migrateAndSeed script failed"
+  echo "migrate script failed"
   exit 1
 fi
 
-# Build-Schema
-# TODO - remove this hack step and replace with lambda service during graphile update
+# Build Graphql Schema
 echo Building Schema
 node $CURRENT_DIR/../pgCatalog/buildPgCatalog.js
 if [ $? != 0 ]; then
   echo "buildPgCatalog failed"
   exit 1
 fi
 
+# Deploy graphql Lambda function with schema
 sls deploy -f graphql
 if [ $? != 0 ]; then
   echo "sls deploy -f graphql failed"
   exit 1
 fi
 
+# If its a new deployment (as indicated by createS3Bucket.js), then seed data
+source $CURRENT_DIR/seed_flag.tmp
```

Comment:
Does this mean it will always seed on a new deployment? Do we have a way to do a new deployment without using the seed data? If we don't it's not a dealbreaker (we can always just clear out the csv), but I'm just curious.

Reply:
Correct, it will always seed on a new deployment. I guess we could make a seeding whitelist if we wanted to. But I didn't see a use case for deploying the backend without any data.

Reply:
I was very happy to be able to deploy the backend without all of the crossings when making my cameras branch

Reply:
Cool, I'll add a config to make seeding optional